### PR TITLE
liveimage-mount: rebased multiple updates

### PR DESCRIPTION
**Pull request by FGrose**
Originally pull request #9 (with comments & responses), then #15, now rebased to master (omitting the OverlayFS feature as it has yet to be adopted).

### CODE REVIEW

File: tools/liveimage-mount [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+# coding: utf-8
+
+# liveimage-mount: Mount a LiveOS at the specified point, and
+#                  enter into a subshell.
 #
-# liveimage-mount: Mount a LiveOS at the specified point, and log
-# into a shell.
-#
-# Copyright 2011, Red Hat  Inc.
+# Copyright 2011, Red Hat, Inc., 2016-12-30, Sugar Labs®
```
Comment (Conan-Kudo) [inline tools/liveimage-mount]: Please move the Sugar Labs copyright to its own line.

**Comment by FGrose** [comment]
@Conan-Kudo re: absolute paths:  Absolute paths void the utility of the PATH environment variable to manage executable programs.  And the default PATH for a Fedora user includes /usr/sbin.

**Comment by Conan-Kudo** [comment]
@FGrose I'm aware that the default path in `bash` and `zsh` include the sbin dirs, but not all shells in Fedora do (for example, `fish` does not). It's hard to rely on it when there are examples of it not being there...

**Comment by Conan-Kudo** [comment]
That being said, if we want to move everything to using the path, that's fine too, but it just hasn't been done that way before...

File: tools/liveimage-mount [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+# liveimage-mount: Mount a LiveOS at the specified point, and
+#                  enter into a subshell.
 #
-# liveimage-mount: Mount a LiveOS at the specified point, and log
-# into a shell.
-#
-# Copyright 2011, Red Hat  Inc.
+# Copyright 2011, Red Hat, Inc.
 # Copyright 2016, Neal Gompa
+# Copyright 2017-01-06, Sugar Labs®
```
Comment (Conan-Kudo) [inline tools/liveimage-mount]: Change this to just only the year, rather than a full date.

**Pull request merged**